### DISCUSSION TITLE
Help adding and linking pages with gohugo-theme-ananke

### DISCUSSION THREAD

**Discussion by john1-design** [community/community#136992] [Programming Help]
### Select Topic Area

Question

### Body

Can you please help me further set up the gohugo-theme-ananke thank you for your time. 

here is my repo, I dont think the pages are created or linked. 

https://github.com/john1-design/gohugo-theme-ananke

**Comment by ebndev** [comment]
Hi @john1-design, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, can you share more details about what you're trying to accomplish? What have you set up so far?
